gammapy/detect/test_statistics.py

```diff
@@ -196,12 +196,12 @@ def compute_maximum_ts_map(ts_map_results):
     Parameters
     ----------
     ts_map_results : list
-        List of `~gammapy.image.SkyMapCollection` objects.
+        List of `~gammapy.image.SkyImageCollection` objects.
 
     Returns
     -------
-    TS : `~gammapy.image.SkyMapCollection`
-        `~gammapy.image.SkyMapCollection` objects.
+    TS : `~gammapy.image.SkyImageCollection`
+        `~gammapy.image.SkyImageCollection` objects.
```

Comment:
Can you improve the documentation of what is returned here?
Duplicating the return type in the description doesn't make sense.
How about this?
```
images : `~gammapy.image.SkyImageCollection`
    Images (ts, niter, amplitude)
```